### PR TITLE
abbr: add page

### PR DESCRIPTION
**Pull request by JohannesPertl**
- [x] The page (if new), does not already exist in the repository.
- [x] The page is in the correct platform directory (`common/`, `linux/`, etc.)
- [x] The page has 8 or fewer examples.
- [x] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#commit-message).
- [x] The page follows the [content guidelines](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#guidelines).
- [x] The page description includes a link to documentation or a homepage (if applicable).

**Version of the command being documented (if known): 3.3.1

### CODE REVIEW

File: pages/linux/abbr.md [hunk 1]
```diff
@@ -0,0 +1,21 @@
+# abbr
+
+> Manages abbreviations for the fish shell.  
+> User-defined words are replaced with longer phrases after they are entered.  
+> More information: <https://fishshell.com/docs/current/cmds/abbr.html>.
+
+- Add a new abbreviation:
+
+`abbr -a -g gco git checkout`
```
Comment (EmilySeville7cfg) [inline pages/linux/abbr.md:9]: It's better to use long options to make tldr pages more readable and memorizable:

```suggestion
`abbr --add gco git checkout`
```

Maybe better to remove `-g` flag to make abbreviation universal and automatically visible in other shell sessions.

**Review by EmilySeville7cfg — CHANGES_REQUESTED** [review]
It's better to use long options to make tldr pages more readable and memorizable.

File: pages/linux/abbr.md [hunk 2]
```diff
@@ -0,0 +1,21 @@
+# abbr
+
+> Manages abbreviations for the fish shell.
```
Comment (navarroaxel) [inline pages/linux/abbr.md:3]:
```suggestion
> Manage abbreviations for the fish shell.
```

File: pages/linux/abbr.md [hunk 3]
```diff
@@ -0,0 +1,21 @@
+# abbr
+
+> Manages abbreviations for the fish shell.  
+> User-defined words are replaced with longer phrases after they are entered.
```
Comment (navarroaxel) [inline pages/linux/abbr.md:4]:
```suggestion
> User-defined words are replaced with longer phrases after they are entered.
```

File: pages/linux/abbr.md [hunk 4]
```diff
@@ -0,0 +1,21 @@
...
+> User-defined words are replaced with longer phrases after they are entered.  
+> More information: <https://fishshell.com/docs/current/cmds/abbr.html>.
+
+- Add a new abbreviation:
+
+`abbr --add gco git checkout`
+
+- Rename an existing abbreviation:
+
+`abbr --rename gco gch`
```
Comment (navarroaxel) [inline pages/linux/abbr.md:13]: I think this is more clear:
```suggestion
`abbr --rename {{old_name}} {{new_name}}`
```

File: pages/linux/abbr.md [hunk 5]
```diff
@@ -0,0 +1,21 @@
+# abbr
+
+> Manages abbreviations for the fish shell.  
+> User-defined words are replaced with longer phrases after they are entered.  
+> More information: <https://fishshell.com/docs/current/cmds/abbr.html>.
+
+- Add a new abbreviation:
+
+`abbr --add gco git checkout`
```
Comment (navarroaxel) [inline pages/linux/abbr.md:9]: Perhaps this better
```suggestion
`abbr --add {{abbreviation_name}} {{command}} {{command_arguments}}`
```

File: pages/linux/abbr.md [hunk 6]
```diff
@@ -0,0 +1,21 @@
...
+
+- Rename an existing abbreviation:
+
+`abbr --rename gco gch`
+
+- Erase an existing abbreviation:
+
+`abbr --erase gco`
```
Comment (navarroaxel) [inline pages/linux/abbr.md:17]:
```suggestion
`abbr --erase {{abbreviation_name}}`
```

File: pages/linux/abbr.md [hunk 7]
```diff
@@ -0,0 +1,21 @@
...
+
+- Import the abbreviations defined on another_host over SSH.:
+
+`ssh another_host abbr -s | source`
```
Comment (navarroaxel) [inline pages/linux/abbr.md:21]: We prefer long arguments if available
```suggestion
`ssh {{host_name}} abbr --show | source`
```

File: pages/linux/abbr.md [hunk 8]
```diff
@@ -0,0 +1,21 @@
...
+
+- Rename an existing abbreviation:
+
+`abbr --rename gco gch`
+
+- Erase an existing abbreviation:
+
+`abbr --erase gco`
+
+- Import the abbreviations defined on another_host over SSH.:
```
Comment (navarroaxel) [inline pages/linux/abbr.md:19]:
```suggestion
- Import the abbreviations defined on another host over SSH:
```

**Review by navarroaxel — APPROVED** [review]

**Review by sbrl — APPROVED** [review]
While that last command demonstrates the functionality of other commands, it's very cool so it can stay :-)

Great job!

**Pull request merged**